.vale/styles/Vocab/OpenShiftDocs/accept.txt

```diff
@@ -3,8 +3,14 @@
 
 [Pp]assthrough
 Assisted Installer
+Control Plane Machine Set
 custom resource
 custom resources
+machine
+machines
 MetalLB
+Nutanix
 Operator
 Operators
+VMware
+vSphere
```

Comment:
Vale doesn't have a problem with `vSphere` but I was trying to make sure it suggests this if the name is mistyped

Reply:
So for VMware or any other non-OCP term, I would strongly suggest making a similar PR on VRH rules. See https://github.com/redhat-documentation/vale-at-red-hat/blob/main/.vale/styles/RedHat/Spelling.yml#L400 for details.

Reply:
Making these kinds of changes on VRH repo is of benefit to all in RH.